front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -58,6 +64,7 @@ export function AgentMessage({
       } = JSON.parse(messageEvent.data);
 
       const event = eventPayload.data;
+      lastEventId.current = eventPayload.eventId;
```

Comment:
FMI why can't it be state ?

Reply:
useRef() is to keep a state in your component that does not trigger rendering and hooks when it changes.
You can use it at rendering time, but changing it won't trigger a re-rendering. Perfect for lastEventId or ressource like a socket.